Makefile.riot

```diff
@@ -1,7 +1,9 @@
 MODULE = tinydtls
 
-CFLAGS += -DDTLSv12 -DWITH_SHA256
```

Comment:
Why is CFLAGS removed=

Reply:
It's already set [by RIOT](https://github.com/RIOT-OS/RIOT/blob/master/pkg/tinydtls/Makefile.include#L12)

Reply:
Thanks